app/views/sensors/add_sensor_reading.haml

```diff
@@ -0,0 +1,13 @@
+= button_tag "Add Sensor Reading manually", :id => 'unhide_sensor_reading_form', :class => "btn btn-primary"
```

Comment:
The normal template is redundant. It is a leftover of me trying to move the function to a single page. The way it is implemented now, only the partial template is needed.